src/06-loops.js

```diff
@@ -151,7 +216,12 @@ function lastIndexFound(array, target) {}
  *  //> [ 7 * 0, 10 * 1, 11 * 2 ]
  *  //> [ 0, 10, 22 ]
  */
-function timesIndex(array) {}
+function timesIndex(array) {
```

Comment:
Remember, this asked for a _new_ `Array` in memory. Here we are returning the same `Array `as our input, but mutated. With a bit stricter testing this likely would not have passed.